sklearn/metrics/tests/test_pairwise.py

```diff
@@ -138,6 +138,23 @@ def test_pairwise_boolean_distance(metric):
             res[np.isnan(res)] = 0
             assert np.sum(res != 0) == 0
 
+    # non-boolean arrays are converted to boolean for boolean
+    # distance metrics with a data conversion warning
+    msg = ("Data was converted to boolean "
+           "for metric %s" % (metric))
+    with pytest.warns(DataConversionWarning, match=msg):
+        pairwise_distances(X, metric=metric)
+
+
+def test_no_data_conversion_warning():
+    # No warnings issued if metric is not a
+    # boolean distance function
```

Comment:
Again, no need to split such comments on multiple lines before the 79 chars limit of PEP8.

Reply:
done